core/admin/ExcAdminSite.py

```diff
@@ -0,0 +1,79 @@
+from django.contrib.admin import AdminSite
+from django.urls import NoReverseMatch, reverse
+from django.apps import apps
+from django.utils.text import capfirst
+
+
+class ExcursionAdmin(AdminSite):
+    index_template = "admin_index.html"
+
+    site_header = 'Excursion Admin'
+    site_title = 'Excursion Admin'
+    index_title = 'Admin Home'
+
+    def _build_app_dict(self, request, label=None):
+        """
+        Build the app dictionary. The optional `label` parameter filters models
+        of a specific app.
+        """
+        app_dict = {}
+
+        if label:
+            models = {
+                m: m_a for m, m_a in self._registry.items()
```

Comment:
I have no clue what `m` and `m_a` is

Reply:
Lol, neither do I. This is copied straight from the overridden method

Reply:
Figured it out. It's the model class, and the admin for that model respectively. I refactored to make it more explicit.